tests/cql/CqlDateTimeOperatorsTest.xml

```diff
@@ -598,7 +603,7 @@
 		<test name="DateTimeDurationBetweenUncertainMultiply">
 			<expression>(days between DateTime(2014, 1, 15) and DateTime(2014, 2))
 				* (days between DateTime(2014, 1, 15) and DateTime(2014, 2))</expression>
-			<output>Interval[ 289, 1936 ]</output>
+			<output>Interval[ 256, 1936 ]</output>
```

Comment:
See `DateTimeDurationBetweenUncertainInterval` for calculation of	`days between DateTime(2014, 1, 15) and DateTime(2014, 2)`, then:
* Uncertainty(16, 44) * Uncertainty(16, 44) = `Uncertainty(256, 1936)`